script/rails-deploy-before-down

```diff
@@ -92,7 +91,17 @@ then
     *****************************************************************
 
 END
-    echo "ENV['RAILS_ENV'] ||= 'production'" > config/rails_env.rb
+fi
+
+RAILS_ENV=$(ruby -r ./config/rails_env.rb -e "puts ENV['RAILS_ENV']" 2> /dev/null)
+if [ "$STAGING_SITE" = "0" ] && [ "$RAILS_ENV" != "production" ] ||
+   [ "$STAGING_SITE" = "1" ] && [ "$RAILS_ENV" == "production" ]; then
+    # STAGING_SITE and RAILS_ENV mismatch - remove line
+    sed -i "/^ENV\['RAILS_ENV'\]/d" config/rails_env.rb 2> /dev/null
```

Comment:
Should it error rather than silently change human-driven config?

Reply:
Error here I was avoiding was when the `config/rails_env.rb` doesn't exist. I could prefix this line with `[ -f config/rails_env.rb ] &&` instead.

Reply:
I'm not sure what makes most sense for this one – happy to go with whichever you think is clearest